OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -832,7 +835,7 @@ public void RemoveInfluence()
 
 		public Activity MoveTo(CPos cell, int nearEnough, Color? targetLineColor = null)
 		{
-			return new Fly(self, Target.FromCell(self.World, cell), targetLineColor: targetLineColor);
+			return new Fly(self, Target.FromCell(self.World, cell), targetLineColor: targetLineColor, nearEnough: nearEnough);
```

Comment:
then this should be `nearEnough: WDist.FromCell(nearEnough)`.

Reply:
Changing the `IMove` interface to accept `WDist` needs to happen at some point, but IMO that is out of scope for this PR.